tests/SdkTest.php

```diff
@@ -92,15 +92,9 @@ public function testAddBreadcrumb(): void
     {
         $breadcrumb = new Breadcrumb(Breadcrumb::LEVEL_ERROR, Breadcrumb::TYPE_ERROR, 'error_reporting');
 
-        /** @var ClientInterface|MockObject $client */
-        $client = $this->createMock(ClientInterface::class);
-        $client->expects($this->once())
-            ->method('addBreadcrumb')
-            ->with($breadcrumb, Hub::getCurrent()->getScope());
-
-        Hub::getCurrent()->bindClient($client);
-
         addBreadcrumb($breadcrumb);
```

Comment:
It's done two lines after this one. I've removed the call on top because the client is no longer involved here.

Reply:
My fault, I was talking about the `addBreadcrumb` method not being called through `Hub::getCurrent()` but what I said didn't make sense since we're testing the global functions